composites/Plugin/SnippetEditor/components/ReplacementVariableEditor.js

```diff
@@ -222,6 +253,43 @@ class ReplacementVariableEditor extends React.Component {
 		this.editor = editor;
 	}
 
+	/**
+	 * Triggers the Draft.js mention plugin suggestions autocomplete.
+	 *
+	 * It does this by inserting the trigger (%) into the editor; replacing the current selection.
+	 * Ensuring the autocomplete shows by adding spaces around the trigger if needed.
+	 *
+	 * @returns {void}
+	 */
+	triggerReplacementVariableSuggestions() {
+		// First remove any selected text.
+		let editorState = removeSelectedText( this.state.editorState );
+
+		// Get the current block text.
+		const selection = editorState.getSelection();
+		const content = editorState.getCurrentContent();
+		const text = getAnchorBlock( content, selection ).getText();
```

Comment:
Content contains the text as well as the selection before and after rendering. But I realize this could be confusing so I'm going to rename all the content to contentState and selection to selectionState.